src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp

```diff
@@ -36,12 +64,11 @@ void CenteredFaceCoordinatesCompute<Dim>::function(
       result->emplace(grid_size);
     }
     evolution::dg::project_tensor_to_boundary(make_not_null(&(result->value())),
-                                              grid_coords, mesh,
+                                              inertial_coords, mesh,
                                               direction.value());
     for (size_t i = 0; i < Dim; ++i) {
-      result->value().get(i) -= excision_sphere.center().get(i);
+      result->value().get(i) -= particle_position.get(i);
```

Comment:
This is missing an `if constexpr (Centered)`, or an error branch that makes sure that this function isn't called for `Centered = false`.

Reply:
gcc complains that this function should be marked no_return for some instantiations. I guess it is true that the struct should have this overload for some template parameters. I was trying to avoid solving this with `Requires` on the struct itself but can do that if you think it's the right thing to do...

Reply:
Old GCC versions are quite bad at that, yea. We usually disable the warning with a pragma diagnostic push/pop for gcc, you'll find examples if you search the code.